docs/reference/objects/product/variant/index.md

```diff
@@ -67,6 +72,8 @@ boolean
 {: .label .fs-1 }
 
 Returns `true` if the variant has unlimited stock.
+- When accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), it will return `true` if the variant has unlimited stock on the given date.
+- When accessed independently or through a product, it will return `true` if the variant has unlimited stock on at least one of the product dates.
```

Comment:
```suggestion
- When accessed independently or through a product, it will return `true` if the variant has unlimited stock on at least one of the upcoming product dates.
```